Enhancement: improve fritzbox proxy perfomance (#2429)
gethomepage/homepage: docs/widgets/services/fritzbox.md

[x] Transmit status information over UPnP
```

> You don't need to provide any credentials. Since that, you should consider using `http` instead of `https` as the requests are significantly faster.

Reviewer: ```suggestion
Credentials are not needed and, as such, you may want to consider using `http` instead of `https` as those requests are significantly faster.
```
